hybrisCustom/custom-ext/bonstore/bonstorestorefront/web/webroot/WEB-INF/config/spring-security-config.xml

```diff
@@ -292,5 +294,19 @@
 	<bean id="rememberMeAuthenticationProvider" class="org.springframework.security.authentication.RememberMeAuthenticationProvider" >
 		<property name="key" value="bonstorestorefront"/>
 	</bean>
+	
+	<bean id="bonstoreLoginFailureHandler" class="org.bonstore.storefront.security.BonstoreLoginFailureHandler" >
```

Comment:
you can use parent to avoid initilizing properties which are defined in the parent class